scripts/r.rgb/r.rgb.py

```diff
@@ -57,13 +57,13 @@ def main():
     expressions = []
     maps = []
     if red:
-        expressions.append("%s = r#${input}" % red)
+        expressions.append("%s = r#${input}" % red)  # noqa: RUF027
         maps.append(red)
     if green:
-        expressions.append("%s = g#${input}" % green)
+        expressions.append("%s = g#${input}" % green)  # noqa: RUF027
         maps.append(green)
     if blue:
-        expressions.append("%s = b#${input}" % blue)
+        expressions.append("%s = b#${input}" % blue)  # noqa: RUF027
         maps.append(blue)
     expr = ";".join(expressions)
     gs.mapcalc(expr, input=input)
```

Comment:
This brings up the larger issue of r.mapcalc syntax in Python. String templates were adopted to be used internally, but it was never mandatory (it can't be) and it was before format function and f-strings and their appeal was closeness to shell/Bash substitution. Having theses RUF ignores here is not nice. It could just switch to f-strings. I'm not sure what to do here in this PR specifically.

Reply:
Can input be safely substituted here without passing the "input" to mapcalc? Otherwise we can keep it as I changed it here, that is to not use an f-string because of the conflict. It's not that bad to say no to a syntax

Reply:
I think it us the same, but on a second thought, this PR is not the place to change the practice.